packages/backend/src/config/config.production.ts

```diff
@@ -22,5 +22,6 @@ export function getProductionConfig(): Config {
       },
       syncBatchSize: getEnv.integer('SYNC_BATCH_SIZE', 6_000),
     },
+    freshStart: getEnv.boolean('FRESH_START', false),
```

Comment:
I don't think we should make dropping the production database that easy :) I suggest keeping it false